### PR TITLE
feat: implemented expo-updates

### PR DESCRIPTION
**Pull request by frankcalise**
# Description

[Trello Card](13-ota-updates-with-expo-update)

- Implements OTA updates
- Added documentation to README about publishing

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features 
- [x] I have run tests and linter

### CODE REVIEW

**Review by frankcalise — COMMENTED** [review]
Add notes

**Review by frankcalise — COMMENTED** [review]
Added a comment about which profile to use, will update docs.

Also, this cookbook pending PR might help wrap your head around the topic if needed:
https://github.com/infinitered/ignite-cookbook/pull/36

File: README.md [hunk 1]
```diff
@@ -99,3 +99,10 @@ _Note: we'll get this more automated with a script in the future_
 5. `yarn start`
 6. Open iOS simulator if you want to watch tests in action
 7. `yarn e2e:ios` (separate terminal, do not stop step 3)
+
+# Publishing updates OTA
+
+1. Create the update branch `eas update --branch [update branch name] --message "info about the update"`
+2. Determine channel where update should be applied. By default, any profile name in `eas.json` is created as a channel in addition to the specified `channel` property
+3. Publish the update branch to the release channel `eas channel:edit [channel name] --branch [branch name]`
+4. Restart the app twice to receive the update (on the first, the app will download any available updates - then apply it on the second)
```
Comment (mazenchami) [inline README.md:108]: this is an interesting now. considering how short of a time we're talking about, is there a way to know if we're waiting on a restart and then prompt the user to restart the app?

Comment (frankcalise) [reply README.md:108]: There are ways to implement finer controls with knowing when the update is downloaded, prompting the user etc. @GantMan advised going with the simple route for now. But to answer your question (and what my workshop will get into as well), yes you can know these things outside of the base set up.

**Review by mazenchami — APPROVED** [review]
🚀

**Review by robinheinze — APPROVED** [review]

**Pull request merged**